libde265/vps.cc

```diff
@@ -186,8 +186,8 @@ if (layer[i].vps_max_dec_pic_buffering == UVLC_ERROR ||
       vps_num_ticks_poc_diff_one = get_uvlc(reader)+1;
       vps_num_hrd_parameters     = get_uvlc(reader);
 
-      if (vps_num_hrd_parameters >= 1024) {
-        assert(false); // TODO: return bitstream error
+      if (vps_num_hrd_parameters < 0 || vps_num_hrd_parameters >= 1024) {
```

Comment:
When get_uvlc() returns -1, this is actually an error condition. In other files (e.g. sps.cc) this is then handled in the READ_VLC() macro. So the preferred why to handle this hind of things is to replace the get_uvlc() calls with a READ_VLC() macro.

Reply:
This is already changed in a later commit (see https://github.com/strukturag/libde265/pull/157/commits/a306aeed626e3ef4581cc223f7ebb169944b0600)